Client/Blogger/src/Components/Footer.jsx

```diff
@@ -0,0 +1,46 @@
+import "../Styles/Footer/Footer.css";
+export default function Footer() {
+  return (
+    <div className="footer" id="footer">
+      <div className="contact-form-wrapper">
+        <form>
+           <label>Feedback</label>
+           <textarea /> 
+
+           <button>
+            Send
+          </button> 
+        </form>
+      </div>
+      <div className="social-media-container">
+        <u>Follow us</u>
+        <div className="media-wrapper">
```

Comment:
For better code organization and reusability, move the SVG code into separate file and import then use it